questions/sort() method.md

```diff
@@ -0,0 +1,11 @@
+# Coding question
+
+## Question
+
+Write a program that initialize an empty list, then populate the list with 20 random integers from 1 to 99. Then write a function to sort those integers in ascending order. Print the result after each step.
+
+Sample:
+
```

Comment:
To create a bit more of a challenge, you might add some restrictions on the kinds of methods that can be used to solve the problem(ex: not being allows to use Collections.sort());